src/config.rs

```diff
@@ -226,25 +384,44 @@ pub struct SyncerServers {
     pub monero_wallet_dir: Option<String>,
 }
 
+#[derive(Deserialize, Serialize, Default, Debug, Clone)]
+#[serde(crate = "serde_crate")]
+pub struct Networked<T> {
```

Comment:
I both hate & love this struct naming xD
I'm guessing you wanted to avoid Network because this here is limited to config? Then might go for `NetworkConfig`? Anyway, keen to get your rationale for this one on Monday.

Reply:
:) yes network is to generic, just wanted to express that it’s a wrapper to “network” a type. NetworkConfig would be fine (if we don’t introduce some [network..] section later) or PerNetwork maybe.